prescreeners/va.html

```diff
@@ -23,7 +23,10 @@ <h1 class="title">Virginians, could you be eligible for help with your food budg
                 class="usa-form"
                 data-state-or-territory="VA"
                 data-use-emergency-allotment="true">
-            <label class="usa-label" for="household_size" id="household-size-label">Household size:</label>
+            <label class="usa-label" for="household_size" id="household-size-label">Household size</label>
+            <div class="label-sub-text">
+              <em>Your household is the people you live with and buy food with. Include children 21 or younger, parents, and spouses if they live with you.</em>
```

Comment:
Guidance on the definition of a SNAP household; based on VA Hunger Solutions fact-sheet: http://vahungersolutions.org/wp-content/uploads/2018/09/SNAP-Household-Fact-Sheet.pdf